doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -2184,23 +2184,23 @@ Response:
 This operation allows to create, update and/or delete several entities in a single batch operation.
 The payload is an object with two properties:
 
-+ `actionType`, to specify the kind of update action to do: either APPEND, APPEND_STRICT, UPDATE,
-  DELETE, or REPLACE.
++ `actionType`, to specify the kind of update action to do: either `append`, `appendStrict`, `update`,
```

Comment:
English lesson ...
Can 'either' be used if there is more than two options?

Reply:
I only change the literals, that part was that way already. However, no problem in changing the surronding text is something is wrong :)

Reply:
I don't know if it's wrong or right ... 
I was hoping Jason Fox would be with us and let us know ... :)

Reply:
https://english.stackexchange.com/questions/170409/whats-the-best-way-to-use-either-on-more-than-two-options

Seems it is correct English.
Still, I find it strange !!!

NTC